testapi.pm

```diff
@@ -1113,7 +1126,7 @@ sub validate_script_output($&;$) {
         bmwqemu::diag("output does not pass the code block:\n$output");
     }
     # abusing the function
-    $autotest::current_test->record_serialresult($output, $res, $output);
+    $autotest::current_test->record_serialresult($output, $res, $output) unless ($args{quiet});
```

Comment:
Looking at the code, and the test run, this is information that might be needed from a test review perspective. Perhaps the quiet parameter should only keep calls beyond this point, but not the final result (I guess this would apply to the *_script functions from the testapi but not to those on the backend...